index.html

```diff
@@ -0,0 +1,612 @@
+						<p class="inflation-paragraph-second"><strong>STILL NOT ENOUGH TIME</strong> There has not been enough time to carry coffee cups from place to place before they get cold. But if all the coffee cups were somehow filled from a single coffee pot, all at the same time, that might explain their even temperature.</p>
+						<p class="inflation-paragraph-second"><strong>INFLATION</strong> solves this problem. The theory proposes that, less than a trillionth of a second after the Big Bang, the universe expanded faster than the speed of light. Tiny ripples in the violently expanding energy field eventually grew into the large-scale structures of the universe.</p>
+						<p class="inflation-paragraph-second"><strong>FLUCTUATION</strong> Astronomers have now detected evidence of these ancient fluctuations in swirls of polarized light in the cosmic background radiation, which is energy left over from the early universe. These are gravitational waves predicted by Einstein.</p>
+						<p class="inflation-paragraph-second"><strong>EXPANSION</strong> Returning to our coffee, imagine a single, central pot expanding faster than light and cooling to an even temperature as it expands. That is something like inflation. And the structure of the universe mirrors the froth and foam of the original pot.</p>
+					</div>
+				</div>
+				<hr class="theory-line">
+				<p class="theory-author">By LARRY BUCHANAN and JONATHAN CORUM</p>
+				<div class="main-article-paragraphs">
+					<p>Dr. Kovac said the chance that the results were a fluke was only one in 10 million.</p>
+					<p>Dr. Guth, now 67, pronounced himself “bowled over,” saying he had not expected such a definite confirmation in his lifetime.</p>
+					<p>“With nature, you have to be lucky,” he said. “Apparently we have been lucky.”</p>
+					<p>The results are the closely guarded distillation of three years’ worth of observations and analysis. Eschewing email for fear of a leak, Dr. Kovac personally delivered drafts of his work to a select few, meeting with Dr. Guth, who is now a professor at Massachusetts Institute of Technology (as is his son, Larry, who was sleeping that night in 1979), in his office last week.</p>
+					<p>“It was a very special moment, and one we took very seriously as scientists,” said Dr. Kovac, who chose his words as carefully as he tended his radio telescopes.</p>
+					<p>Andrei Linde of Stanford, a prolific theorist who first described the most popular variant of inflation, known as chaotic inflation, in 1983, was about to go on vacation in the Caribbean last week when Chao-Lin Kuo, a Stanford colleague and a member of Dr. Kovac’s team, <a href="#">knocked on his door with a bottle of Champagne</a> to tell him the news.</p>
+					<div class="video">
+            <iframe allowfullscreen height="335" src="https://www.youtube.com/embed/ZlfIVEy_YOA" width="580"></iframe>
```

Comment:
- [ ]  put height and width selectors in a class or Id in your CSS file as no inline styling is permitted in the HTML